src/index.d.ts

```diff
@@ -7,12 +7,11 @@ import React from 'react';
 import Sendbird from 'sendbird';
 
 import AppProps from './smart-components/App/types';
+import * as SendbirdTypes from './types';
```

Comment:
Any specific reason for this change or just preference?

Reply:
UserListQuery had not been exported. So there was a type warning(or error) "Can't find module UserListQuery" stuff